web/modules/custom/rdf_entity/rdf_entity.module

```diff
@@ -2,6 +2,152 @@
 
 /**
  * @file
- * Contains Drupal\rdf_entity\rdf_entity.module
+ * Main functions and hook implementations of the RDF Entity module.
  */
 
+use \Drupal\Core\Form\FormStateInterface;
+use \Drupal\field_ui\Form\FieldStorageConfigEditForm;
+use \Drupal\field\Entity\FieldStorageConfig;
+use \Drupal\Core\Render\Element;
+use \Drupal\rdf_entity\RdfInterface;
+/**
+ * Implements hook_form_alter().
+ */
+function rdf_entity_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id != 'field_storage_config_edit_form') {
+    return;
+  }
+  // Only add mapping form element to rdf entities fields.
+  if ($form_state->get('entity_type_id') != 'rdf_entity') {
+    return;
+  }
+  $form_obj = $form_state->getFormObject();
+  /** @var FieldStorageConfig $entity */
+  $entity = $form_obj->getEntity();
+  $schema = $entity->getSchema();
+  $form['rdf_mapping'] = array(
+    '#type' => 'details',
+    '#title' => t('Rdf field mapping'),
+    '#description' => t('This field uses a Sparql backend. Please map the fields to their corresponding rdf properties.'),
+    '#weight' => 99,
+  );
+  foreach ($schema['columns'] as $column => $column_desc) {
+    $description = $column_desc['description'] . "<br>";
```

Comment:
On this line I'm getting the following notice if I visit `admin/structure/rdf_type/manage/agent/overview/fields/rdf_entity.agent.field_name/storage`:

> Notice: Undefined index: description in rdf_entity_form_alter() (line 36 of modules/custom/rdf_entity/rdf_entity.module).